config/manager/e2e-manager.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         args:
         - --enable-leader-election
         - --pprof
+        - --remoteConfigEnabled
```

Comment:
It would be better to have this rc-specific e2e-manager.yaml as a separate file in `config/manager`. I'll make an update in a separate PR to support this.

Reply:
Adding support here: https://github.com/DataDog/datadog-operator/pull/1294